testing/zerotier-one/APKBUILD

```diff
@@ -1,38 +1,48 @@
-# Contributor: Kyle Parisi <kyleparisi@gmail.com>
-# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+# Contributor: ZeroTier, Inc. <contact@zerotier.com>
+# Maintainer: Adam Ierymenko <adam.ierymenko@zerotier.com>
+
+## Thanks to Kyle Parisi <kyleparisi@gmail.com> for doing the original package and getting it into testing
```

Comment:
There can be multiple contributors, so you can simply keep his entry together with the ZeroTier, Inc. one. Also, did ZeroTier, Inc. themselves actually contribute to this APKBUILD?